integration_tests/sdk/data_integration_tests/conftest.py

```diff
@@ -60,3 +64,9 @@ def flow_manager(client, flow_name, engine):
     as it is mostly a mechanism by which data can be saved.
     """
     return FlowManager(client, flow_name, engine)
+
+
+def pytest_sessionfinish(session, exitstatus):
+    if not hasattr(session.config, "workerinput"):
+        client = aq.Client(*get_aqueduct_config())
+        delete_all_flows(client)
```

Comment:
Why do we have two copies of this method? Shouldn't it just go in the parent directory's `conftest.py` once? The subfolders should inherit conftest.py hooks from the parent directories.

Reply:
Hmmm it didn't work when I call that in parent directory, so I put them here